components/elm/bld/namelist_files/namelist_definition.xml

```diff
@@ -298,8 +298,15 @@ Toggle to turn on the FATES
 </entry>
 
 <entry id="fates_spitfire_mode" type="integer" category="physics"
-        group="elm_inparm" valid_values="0,1" >
-Set to 0 to keep fire off. Set to 1 to turn on fates spitfire module (only relevant if FATES is being used).
+        group="elm_inparm" valid_values="0,1,2,3,4,5", value=".false." >
```

Comment:
should the type on value match valid_values? ie, should this be 0 or 1?

Reply:
Good catch @rgknox.  This will need to be changed on the clm side as well.